client/a8c-for-agencies/sections/overview/body/events/hooks/use-upcoming-events.tsx

```diff
@@ -13,50 +12,25 @@ export const useUpcomingEvents = () => {
 	return useMemo( () => {
 		const eventsData: UpcomingEventProps[] = [
 			{
-				id: 'pressable-webinar-2025-11-06',
+				id: 'a4a-partner-survey-2025-12-03',
 				date: {
-					from: moment( '2025-11-06' ),
-					to: moment( '2025-11-06' ),
+					from: moment( '2026-01-01' ),
+					to: moment( '2026-01-01' ),
 				},
-				displayDate: translate( 'Thursday, November 6, 9:00-10:00 AM PT (4:00-5:00 PM UTC)' ),
-				title: translate( 'The Pressable advantage for Automattic for Agencies partners' ),
-				subtitle: translate( 'Automattic for Agencies and Pressable' ),
+				displayDate: translate( 'Open until January 1, 2026' ),
+				title: translate( 'Automattic for Agencies Partner Survey' ),
+				subtitle: translate( 'Automattic for Agencies' ),
 				descriptions: [
 					translate(
-						"Ready to scale your agency without stretching your team thin? Join us for an exclusive live session where we'll reveal how top-performing agencies are using Pressable and Automattic for Agencies to drive faster client growth, increase recurring revenue, and hit year-end goals effortlessly."
+						"We invite you to share your input in our short Automattic for Agencies Partner Survey. Your feedback will help us better understand your experience with Automattic for Agencies and the products you use across our ecosystem. The survey is anonymous, and your insights will guide how we shape next year's incentives, tools, and product improvements to create more value for your agency and clients."
```

Comment:
We should be using curly quotes: pgz0xU-1S-p2

```suggestion
						"We invite you to share your input in our short Automattic for Agencies Partner Survey. Your feedback will help us better understand your experience with Automattic for Agencies and the products you use across our ecosystem. The survey is anonymous, and your insights will guide how we shape next year’s incentives, tools, and product improvements to create more value for your agency and clients."
```

Reply:
Thanks for catching this.